.github/workflows/deploy-all-staging-accounts.yml

```diff
@@ -28,28 +28,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        short_helplines:
-          [
-            'CA',
-            'CL',
-            'PH',
-            'ET',
-            'JM',
-            'MW',
-            'IN',
-            'RO',
-            'BR',
-            'ZA',
-            'CO',
-            'TH',
-            'UK',
-            'HU',
-            'ZM',
-            'ZW',
-            'PL',
-            'MT',
-            'NZ',
```

Comment:
NZ is missing in the update, and the format should be the same. Be sure to add the comma at the end or else the lint test will fail.